src/udp/udp.c

```diff
@@ -65,6 +65,8 @@ struct udp_sock {
 	udp_recv_h *rh;      /**< Receive handler             */
 	udp_error_h *eh;     /**< Error handler               */
 	void *arg;           /**< Handler argument            */
+	struct re_fhs *fhs;
+	struct re_fhs *fhs6;
 	re_sock_t fd;        /**< Socket file descriptor      */
 	re_sock_t fd6;       /**< IPv6 socket file descriptor */
```

Comment:
I think we can remove `fd6` now, so that there is only one socket ...

Reply:
After some research, looks like windows needs separate sockets by default. Dual-Stack needs a explicit `setsockopt` IPV6_V6ONLY set to zero:

https://learn.microsoft.com/en-us/windows/win32/winsock/dual-stack-sockets#creating-a-dual-stack-socket

Will try to create a PR.

Reply:
https://github.com/baresip/re/pull/911